libs/src/services/creatorNode/index.js

```diff
@@ -713,6 +713,17 @@ class CreatorNode {
           }
         }
 
+        // if the content node returns an invalid auth token error, disconnect and reconnect
+        if (e.response && e.response.data && e.response.data.error && e.response.data.error.includes('Invalid authentication token')) {
```

Comment:
```suggestion
        if (resp.data && resp.data.error && resp.data.error.includes('Invalid authentication token')) {
```
defined earlier on L697